deploy/v1.1.0/avalanche.ts

```diff
@@ -0,0 +1,102 @@
+  } = await getNamedAccounts()
+
+  // ===================================================
+  // Deploy Contracts
+  // ===================================================
+
+  const aaveUsdcYieldSourceResult = await deployAndLog('ATokenYieldSource', {
+    from: deployer,
+    args: [
+      aUSDC,
+      aaveIncentivesController,
+      aaveLendingPoolAddressesProviderRegistry,
+      TOKEN_DECIMALS,
+      "PTaUSDCeY",
+      "PoolTogether aUSDC Yield",
+      executiveTeam
+    ]
+  })
+  const yieldSourcePrizePoolResult = await deployAndLog('YieldSourcePrizePool', { from: deployer, args: [deployer, aaveUsdcYieldSourceResult.address] })
+  const ticketResult = await deployAndLog('Ticket', { from: deployer, args: ["Ticket", "TICK", TOKEN_DECIMALS, yieldSourcePrizePoolResult.address] })
+  const prizeTierHistoryResult = await deployAndLog('PrizeTierHistory', { from: deployer, args: [deployer] })
+  const drawBufferResult = await deployAndLog('DrawBuffer', { from: deployer, args: [deployer, DRAW_BUFFER_CARDINALITY] })
+  const prizeDistributionBufferResult = await deployAndLog('PrizeDistributionBuffer', { from: deployer, args: [deployer, PRIZE_DISTRIBUTION_BUFFER_CARDINALITY] })
+  const drawCalculatorResult = await deployAndLog('DrawCalculator', { from: deployer, args: [ticketResult.address, drawBufferResult.address, prizeDistributionBufferResult.address] })
+  const prizeDistributorResult = await deployAndLog('PrizeDistributor', { from: deployer, args: [executiveTeam, ticketResult.address, drawCalculatorResult.address] })
+  const prizeSplitStrategyResult = await deployAndLog('PrizeSplitStrategy', { from: deployer, args: [deployer, yieldSourcePrizePoolResult.address] })
+  const reserveResult = await deployAndLog('Reserve', { from: deployer, args: [deployer, ticketResult.address] })
+  await deployAndLog('DrawCalculatorTimelock', { from: deployer, args: [deployer, drawCalculatorResult.address] })
+
+  const prizeDistributionFactoryResult = await deployAndLog('PrizeDistributionFactory', {
+    from: deployer,
+    args: [
+      deployer,
+      prizeTierHistoryResult.address,
+      drawBufferResult.address,
+      prizeDistributionBufferResult.address,
+      ticketResult.address,
+      PRIZE_DISTRIBUTION_FACTORY_MINIMUM_PICK_COST // 1 USDC
+    ]
+  })
+  await deployAndLog('EIP2612PermitAndDeposit', { from: deployer })
+  const prizeFlushResult = await deployAndLog('PrizeFlush', { from: deployer, args: [deployer, prizeDistributorResult.address, prizeSplitStrategyResult.address, reserveResult.address]})
+  const receiverTimelockAndPushRouterResult = await deployAndLog('ReceiverTimelockAndPushRouter', { from: deployer, args: [deployer, drawBufferResult.address, prizeDistributionFactoryResult.address, drawCalculatorResult.address]})
+
+  // ===================================================
+  // Configure Contracts
+  // ===================================================
+
+  await pushDraw48()
```

Comment:
Should this be changed to the latest draw on Ethereum Mainnet?

Reply:
This is the latest configuration that was pushed to the PrizeTierHistory at draw 48.

The first push to the DrawBuffer will determine at which draw id this prize pool will start